src/backend/web/handlers/event.py

```diff
@@ -129,13 +131,30 @@ def event_detail(event_key: EventKey) -> Response:
         middle_value += 1
     teams_a, teams_b = team_and_medias[:middle_value], team_and_medias[middle_value:]
 
-    oprs = (
-        [i for i in event.matchstats["oprs"].items()]
-        if (event.matchstats is not None and "oprs" in event.matchstats)
-        else []
-    )
-    oprs = sorted(oprs, key=lambda t: t[1], reverse=True)  # sort by OPR
-    oprs = oprs[:15]  # get the top 15 OPRs
+    num_matchstats = 15
+    matchstats = collections.OrderedDict()
+    if event.matchstats is not None:
+        matchstats["OPRs"] = sorted(
+            event.matchstats["oprs"].items(), key=lambda t: -t[1]
```

Comment:
Can we replace these strings with `StatType.XXX.value`?

Reply:
For `event.matchstats["oprs"]` and the like, yes. For `matchstats["OPRs"]`, it will affect the template:

![3o9A4pV](https://user-images.githubusercontent.com/7595639/114792478-72944500-9d56-11eb-908f-ae801d99c291.png)